azure-monitor-opentelemetry/azure/monitor/opentelemetry/_vendor/v0_38b0/opentelemetry/instrumentation/psycopg2/version.py

```diff
@@ -1,15 +0,0 @@
-# Copyright The OpenTelemetry Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-__version__ = "0.38b0"
```

Comment:
Should we remove package.py as well?

Reply:
What would really be the purpose of this?

Reply:
We don't want to have unnecessary files in our vendored folders since these will be every growing. If we can just keep everything within __init__.py that would be ideal. version.py is only really used in README files anyways.

Reply:
Confirmed version is used in the vendored code.